uvm/js/common/util/Util.js

```diff
@@ -884,6 +884,23 @@ Ext.define('Ung.util.Util', {
         return dots.join('.');
     },
 
+    /**
+     * Loop through the stored records to see if the passed ip address is already used
+     * @param  string ip      IP address to be checked.
+     * @param  int store      ExtJs store of records
+     * @param  int dataIndex  ExtJs dataIndex property of records
+     * @return boolean        Returns true if ip already present in dataIndex field of store record.
+     */
+    isAddrUsed: function(ip, store, dataIndex) {
+        var ret = false;
+        store.each(function(record,idx) {
+            if( record.get(dataIndex) == ip ){
+                ret = true;
```

Comment:
Can we `break` from here so that we don't loop through remaining entries ? May have to write a traditional for loop to achieve this.

Reply:
Or may be just `return true` if that works in `store.each()`

Reply:
each() is a consumer function so it by default returns undefined. even if we add return statement loop will continue.
We can't break the forEach loop, there are ugly ways to do so by throwing exception and catching outside loop. but i don't think we should use that

Reply:
Okay.